src/excmds.ts

```diff
@@ -1080,6 +1079,25 @@ export async function buffer(index: number | '#') {
 
 // {{{ SETTINGS
 
+/** Equivalent to vim's `:command`. Maps one ex-mode command to another.
+ * 
+ * Examples:
+ *  - `command t tabopen`
+ *  - `command tn tabnext_gt`
```

Comment:
Need to define semantics more strictly:

 1. What happens if you try to alias an existing command?
 2. Are aliases expanded recursively?
 3. What happens if a user aliases a command and then we add it as a builtin afterwards?

BTW, this is not equivalent to vim. :command in vim requires the defined command to start with a capital letter (to avoid clashes with builtin commands) and has a bunch more features to do with argument and range processing...

We can use the name, but we should say it's similar, not equivalent. We may also want to restrict user-added aliases to capital letters.

We also need some way to remove an alias. vim uses `:comclear`